workloads/network-perf/common.sh

```diff
@@ -132,68 +105,29 @@ export_defaults() {
 }
 
 deploy_operator() {
-  log "Removing benchmark-operator namespace, if it already exists"
-  oc delete namespace benchmark-operator --ignore-not-found
-  log "Cloning benchmark-operator from branch ${operator_branch} of ${operator_repo}"
+  deploy_benchmark_operator ${OPERATOR_REPO} ${OPERATOR_BRANCH}
```

Comment:
can we directly call `deploy_benchmark_operator` instead of `deploy_operator` from run_uperf.sh scripts

Reply:
That's right, however as it needs some common tweaking after deploying the operator I left this function.